app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 class="mt-4 text-center">Public Recipes</h1>
+
+<div id="recipes">
+  <% @public_recipes.each do |recipe| %>
+    <div id="<%= dom_id recipe %>" class="d-flex border border-dark p-3 mb-4">
+      <div class="d-flex flex-column justify-content-between">
+        <p>
+          <strong>Name:</strong>
+          <%= recipe.name %>
+        </p>
+        <span>By <%= recipe.user.name %></span>
+      </div>
+
+      <div class="ms-5">
+        <p>
+          <strong>Description:</strong>
+          <%= recipe.description %>
+        </p>
+        <p>
+          <strong>Preparation time:</strong>
+          <%= recipe.preparation_time %>
+        </p>
+
+        <%# <p>
+          <strong>Cooking time:</strong>
+          <%= recipe.cooking_time %
+        </p> %>
+
+    <%# 
+        <p>
+          <strong>Public:</strong>
+          <%= recipe.public %
+        </p> %>
```

Comment:
Leaving unused code as comments in your codebase is generally not recommended. It can lead to confusion, increase code clutter, and make code maintenance more difficult. Kindly remove all commented code from your codebase to make it look more professional. :+1: